composer.json

```diff
@@ -34,7 +34,8 @@
         "symfony/console":  "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.0"
+        "phpunit/phpunit":    "~5.0",
+        "mikey179/vfsStream": "^1.6"
```

Comment:
Yes, but the primary `vfsStream` usage is to verify that things are *written* somewhere, whilst keeping the real FS clean ;-)